code/modules/reagents/reagent_containers/mk2.dm

```diff
@@ -0,0 +1,379 @@
+#define HYPO_SPRAY 0
+#define HYPO_INJECT 1
+
+#define WAIT_SPRAY 25
+#define WAIT_INJECT 25
+#define SELF_SPRAY 15
+#define SELF_INJECT 15
+
+#define DELUXE_WAIT_SPRAY 20
+#define DELUXE_WAIT_INJECT 20
+#define DELUXE_SELF_SPRAY 10
+#define DELUXE_SELF_INJECT 10
+
+#define COMBAT_WAIT_SPRAY 0
+#define COMBAT_WAIT_INJECT 0
+#define COMBAT_SELF_SPRAY 0
+#define COMBAT_SELF_INJECT 0
+
+//A vial-loaded hypospray. Cartridge-based!
+/obj/item/hypospray/mkii
```

Comment:
This is not under reagent_containers

Reply:
The Vials? Those i'd agree with that but the MK.2 isnt' a container.

Reply:
Why isn't it a container. It contains reagents does it not?

Reply:
You fill vials with the reagents, then load the vials into the MK.2 So you can quick swap chems when you need too.